homeassistant/components/homekit/strings.json

```diff
@@ -8,18 +8,26 @@
             },
             "init": {
                 "data": {
-                    "include_domains": "[%key:component::homekit::config::step::user::data::include_domains%]"
+                    "include_domains": "[%key:component::homekit::config::step::user::data::include_domains%]",
+                    "filter_by_excluding": "Include entities not specifically excluded"
                 },
-                "description": "Entities in the \u201cDomains to include\u201d will be bridged to HomeKit. You will be able to select which entities to exclude from this list on the next screen.",
+                "description": "Entities in the \u201cDomains to include\u201d will be bridged to HomeKit. You will be able to select which entities are bridged from this list on the next screen.",
                 "title": "Select domains to bridge."
             },
             "exclude": {
                 "data": {
                     "exclude_entities": "Entities to exclude"
                 },
-                "description": "Choose the entities that you do NOT want to be bridged.",
+                "description": "Choose the entities that you do NOT want to be bridged. New entities will be automatically included.",
```

Comment:
```suggestion
                "description": "Choose the entities that you do NOT want to be bridged. New entities will be automatically included, the next the bridge is reloaded.",
```

Reply:
Are you sure only they turn up at reload? I'm convinced I've seen new homekit stuff turning up as other matching entities are registered inside HASS while running.

Reply:
If you restart or modify the options flow, we reload the config entry.

Reply:
Yes, but the underlying homekit filter will still include new items that meet the criteria, no?

The frustrating use case I'm trying to solve here is:

- Setup homekit for media_player, and exclude all the media players other than the one you want
- Add a new Apple TV
- Suddenly there's extra media entities showing up in HomeKit because the original filter didn't exclude them.

...that happens without restarting anything, IIRC.

Reply:
> Yes, but the underlying homekit filter will still include new items that meet the criteria, no?
> 

Yes, when the config entry is reloaded anything that is in the state machine that matches the filter will get picked up.

> The frustrating use case I'm trying to solve here is:
> 
> * Setup homekit for media_player, and exclude all the media players other than the one you want
> * Add a new Apple TV
> * Suddenly there's extra media entities showing up in HomeKit because the original filter didn't exclude them.
> 
I understand the use case you are solving for, thank you for adding additional optionality.

> ...that happens without restarting anything, IIRC.

The only way they should appear without restarting is if the options flow was used (on a non-imported entry) as there is a reload at the end which calls this listener in `__init__.py`
```
async def _async_update_listener(hass: HomeAssistant, entry: ConfigEntry):
    """Handle options update."""
    if entry.source == SOURCE_IMPORT:
        return
    await hass.config_entries.async_reload(entry.entry_id)

```